Tests/InkTests/HTMLTests.swift

```diff
@@ -118,7 +233,16 @@ extension HTMLTests {
             ("testTopLevelSelfClosingHTMLElement", testTopLevelSelfClosingHTMLElement),
             ("testInlineSelfClosingHTMLElement", testInlineSelfClosingHTMLElement),
             ("testTopLevelHTMLLineBreak", testTopLevelHTMLLineBreak),
-            ("testHTMLComment", testHTMLComment)
+            ("testHTMLComment", testHTMLComment),
+            ("testMarkdownHeadingInsideHTML", testMarkdownHeadingInsideHTML),
+            ("testMarkdownImageInsideHTML", testMarkdownImageInsideHTML),
+            ("testMarkdownListInsideHTML", testMarkdownListInsideHTML),
+            ("testMarkdownBeforeHTML", testMarkdownBeforeHTML),
+            ("testMarkdownAfterHTML", testMarkdownAfterHTML),
+            ("testMultipleMarkdownInsideHTML", testMultipleMarkdownInsideHTML),
+            ("testHTMLWithDoubleNewline", testHTMLWithDoubleNewline),
```

Comment:
```suggestion
            ("testHTMLWithDoubleNewline", testHTMLWithDoubleNewline),
            ("testUnclosedHTMLWithDoubleNewline", testUnclosedHTMLWithDoubleNewline),
```

Add extra text case.

Reply:
You don't seem to have included the test. I'll add it